src/peer.js

```diff
@@ -147,7 +147,7 @@ class Peer extends EventEmitter {
 
   ping (cb) {
     var start = Date.now()
-    var nonce = crypto.pseudoRandomBytes(8)
+    var nonce = crypto.randomBytes(8)
```

Comment:
Ok, strange. Yesterday I moved from MEAN.io to preboot/angular-webpack and now pseudoRandomBytes works. Maybe newer webpack version...